qiskit/primitives/base_sampler.py

```diff
@@ -71,17 +71,13 @@
     theta3 = [0, 1, 2, 3, 4, 5, 6, 7]
 
     with Sampler(circuits=[pqc, pqc2], parameters=[pqc.parameters, pqc2.parameters]) as sampler:
-        result = sampler(parameters=[theta1, theta2, theta3], circuits=[0, 0])
+        result1 = sampler([0, 0], [theta1, theta2])
 
         # result of pqc(theta1)
         print([q.binary_probabilities() for q in result[0].quasi_dists])
 
         # result of pqc(theta2)
         print([q.binary_probabilities() for q in result[1].quasi_dists])
```

Comment:
I suggest adding back an example of pqc2.

```suggestion
        print([q.binary_probabilities() for q in result[1].quasi_dists])
        
        # result of pqc2(theta3)
        print([q.binary_probabilities() for q in result[2].quasi_dists])
        
```